viz-lib/src/visualizations/table/utils.js

```diff
@@ -78,24 +79,10 @@ export function prepareColumns(columns, searchInput, orderBy, onOrderByChange) {
               {column.title}
             </div>
           </Tooltip>
-          <span className="ant-table-column-sorter">
-            <div className="ant-table-column-sorter-inner ant-table-column-sorter-inner-full">
-              <Icon
-                className={`ant-table-column-sorter-up ${isAscend ? "on" : "off"}`}
-                type="caret-up"
-                theme="filled"
-              />
-              <Icon
-                className={`ant-table-column-sorter-down ${isDescend ? "on" : "off"}`}
-                type="caret-down"
-                theme="filled"
-              />
-            </div>
-          </span>
```

Comment:
Moved this to the built-in sorter.

Reply:
Seems now it's somewhat broken in multi-column mode - all sorters are visible, but only the most left one show a blue arrow:

![image](https://user-images.githubusercontent.com/12139186/90389957-2ee81f00-e093-11ea-89fd-f20326f3cc4b.png)
![image](https://user-images.githubusercontent.com/12139186/90390194-969e6a00-e093-11ea-83ee-c6e2850f114d.png)

Reply:
Fixed in https://github.com/getredash/redash/pull/5068/commits/a6eb1e2410d6eacf4b814d6ff89021e2fa54314f